mithril-infra/assets/ssh_keys

```diff
@@ -1,9 +1,5 @@
 curry:ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAvDxe/uY9xMQgc7OLyej2AkvTQ3FVtFomknmLSrjUTehFeSjByn/3tziZmUgSLfKl9bLyyROFF0qqlc4i1AEjUdxV7Kle9wCf+yYPUuhUTWzzJSLT4RouyXViPQJ1advQddJgD6DRJAQlmh0wf6neSQMDfNKKA7FXXWXYzBG9Gitp+oVMVOq4JyiTNObNl3Fq2/gkERAFxwG53PFiTnskJoh1kkSVzXtQmfBskdvESSL/ubFc0RnfyfFSGQ0E1Q389fhGeH1nmU92wYW52CA95vz7l4e4hVpCk1UCGr9GX/ScdDY0OC/sNuyK468AAMgk1HkITGVcc8/LXRvohylIHQ== abailly@oqube.com
 root:ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAvDxe/uY9xMQgc7OLyej2AkvTQ3FVtFomknmLSrjUTehFeSjByn/3tziZmUgSLfKl9bLyyROFF0qqlc4i1AEjUdxV7Kle9wCf+yYPUuhUTWzzJSLT4RouyXViPQJ1advQddJgD6DRJAQlmh0wf6neSQMDfNKKA7FXXWXYzBG9Gitp+oVMVOq4JyiTNObNl3Fq2/gkERAFxwG53PFiTnskJoh1kkSVzXtQmfBskdvESSL/ubFc0RnfyfFSGQ0E1Q389fhGeH1nmU92wYW52CA95vz7l4e4hVpCk1UCGr9GX/ScdDY0OC/sNuyK468AAMgk1HkITGVcc8/LXRvohylIHQ== abailly@oqube.com
-curry:ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBflPLBodOBVCheUWlJTzp0naIIqYS6wYuvDzJxpw7Ne
-curry:ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMw0R8ZkJEPFK3d/g9KxkhJx/jTzN2zNOrUVoi+87QMt
-curry:ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIK5pnmoxuIEwDlIgnr3IUAuVpLgoD9rxcz3gG7W48+An
-curry:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDQTUfb74hSiFrurQ5QjpxVa47lEI2Clf/TU2fmxsMkavZh59LEtKgbc2Nz2AoNyO8he2Ec8Kf76GzZYP7SFG49MnMAkpb8QmeAce9soK4Tbk3jzsXfcleXTOf5elNcgxyROb5pT7rgv/xaZEz1f4cy+kT6iyujkQbMBRBVQkidqFZdzD7HRo4PmoGQ3BvQsDVV36l11rM+bVO48k7hzyU2/B/77s8JS6urnk6FSKtKgjWNvr4ntqOu1bivHELmddT4sb1/7f6vNAODucfWVFAOlShiZ1wT1FOI6WJ48TMGA4CMBVGryC8RqGpq4QiRQlRCPoj9Pf+/Yjr5H8PaH8qvyBX9F818/6m19Ai8uierx6ce4OU7vDQ+DD8hjdXDG4cvK9ZPngQfHGeKn9XiO3QKdLGrIGfQGQGxRZDsEx7RGtBLrslrZpY5qY63OyNxyaAUTUYfR8Rs+V1p3sixdq6AB7Dv81mXQ5eNqdk3554U73bdoSPWNBdI188ftdmcl5pc97UlxFRLhrwk8XqeVo3eHfTb7HrG6/4+f+d+zMGKC51YNs6JiiTyviSk4KlCq4QULUKQ09V+W/s1JhSmKYR0EOv/xkbfanhuuBZu8BGFzu230+Awbhg9e+Mpioy0961apHuAtkUtDATO2QydOZwMsPweWRoz79Lf7gJ+PWtOvQ== Key for terraform provisioning through GitHub actions
```

Comment:
Could be removed since Terraform ssh key is handled by the `Prepare service account credentials` of the `ci.yml` pipeline.